odh-dashboard/base/deployment.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: rhods-dashboard
+  name: odh-dashboard
 spec:
-  replicas: 5
+  replicas: 2
```

Comment:
hi @lucferbux any specific reason for reducing the replica set count ?

Reply:
Yes, we are not changing the replicas, the overlay will keep it as 5, we are syncing that file with upstream, and in odh we deploy 2 replicas.

Reply:
the "real" value for rhods is set in odh-dashboard/overlays/rhods/deployment.yaml